### PR TITLE
Removed binary .sqlite file and added to .gitignore.

### PR DESCRIPTION
**Pull request by donigian**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ venv
 
 #pycharm auto generated files
 .idea
+
+students/data.sqlite
```
Comment (donigian) [inline .gitignore:7]: why did u remove the binary file?

**Comment by JMercadoPCC** [comment]
So thats what you do?

@nerd7023 nope.

**Comment by nerd7023** [comment]
Do you know the Muffin Man?

File: .gitignore (hunk 1 above)
Comment (amritpal360) [inline .gitignore:7]: hello

Comment (joerenzullo) [inline .gitignore:7]: The quick brown fox jumps over the lazy dog

**Comment by danielchambers** [comment]
hey

File: .gitignore (hunk 1 above)
Comment (JHSegura) [inline .gitignore:7]: Done

**Comment by kelvinsilva** [comment]
I LOVE MANGOS AND PEACHES

**Comment by ryanvdv** [comment]
The muffin man? Who lives on Drury Lane?

**Comment by davidtea** [comment]
Hi

File: .gitignore (hunk 1 above)
Comment (daveryaaziem) [inline .gitignore:7]: I can commit this way

Comment (daveryaaziem) [inline .gitignore:7]: @jorenzillo its the quick brown fox.... otherwise there's no Q ;)

Comment (joerenzullo) [inline .gitignore:7]: I always mess that up. :P

**Pull request merged**